src/main/java/core/basesyntax/service/WriterToCsv.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface WriterToCsv {
```

Comment:
The interface name 'WriterToCsv' suggests that it is specifically for writing to CSV files, but it's better to have a generic name such as 'WriterService' so that the implementation can be reused for different file types if needed.